apps/client/src/components/organisms/panel/Panel.component.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactElement, useEffect, useState } from 'react';
+import { Description, Text } from '../..';
+import { ReactComponent as ArrowHeadTop } from 'pixelarticons/svg/chevron-up.svg';
+import cs from 'classnames';
+
+interface PanelProps {
+  title: string;
+  children: ReactElement<typeof Description>[];
+}
+
+export function Panel({ title, children }: PanelProps): ReactElement {
+  const [isCollapse, setCollapse] = useState(true);
+  useEffect(() => {
+    const element = document.getElementById('scrollPanel');
+    const handleScroll = (): void => {
+      element?.scrollTo({ top: 0, behavior: 'smooth' });
+    };
+    handleScroll();
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, [isCollapse]);
+
+  //set min sizes to panel
+
+  return (
+    <div
+      className={cs(
+        { 'h-40 pt-6': isCollapse },
+        { 'h-2/3 pt-20 pb-24': !isCollapse },
+        'flex flex-col',
+        'absolute top-5 right-5',
+        'px-10',
+        'w-1/3 max-w-xl max-h-[1200px] rounded-md bg-bg-brown',
+        'border-4 border-bg-brown-100',
+        'transition-all duration-300',
+      )}
+      onClick={(): void => setCollapse(preValue => !preValue)}>
+      <div
+        className={cs(
+          { 'gap-20 overflow-y-hidden': isCollapse },
+          { 'overflow-y-scroll scrollbar': !isCollapse },
+          'flex flex-col pr-5',
+          'transition-all duration-300',
+          'gap-5',
+        )}
+        id='scrollPanel'>
```

Comment:
Why do we need a `id` here

Reply:
The id is used to scroll the panel to the top when collapsing.

 ```
   useEffect(() => {
    const element = document.getElementById('scrollPanel');
    const handleScroll = (): void => {
      element?.scrollTo({ top: 0, behavior: 'smooth' });
    };
    handleScroll();
    return () => {
      window.removeEventListener('scroll', handleScroll);
    };
  }, [isCollapse]);
  ```